runtime-image/test/test.ts

```diff
@@ -190,23 +187,23 @@ describe('runtime image', () => {
   });
 });
 
-function buildDocker(dir: string, tag: string): Promise<any> {
+function buildDocker(dir: string, tag: string): Promise<{}> {
   const tarStream = tar.pack(dir);
-  return new Promise<any>((resolve, reject) => {
-    DOCKER.buildImage(tarStream, {t : tag}, (err1, stream) => {
+  return new Promise<{}>((resolve, reject) => {
+    DOCKER.buildImage(tarStream, {t: tag}, (err1, stream) => {
```

Comment:
The dockerode API appears to support promises:  
https://github.com/apocas/dockerode/blob/master/lib/docker.js#L285

Can we use that instead of wrapping?

Reply:
Yes. Updated.